epub33/core/index.html

```diff
@@ -4092,348 +4092,376 @@ <h4>Prefixed Properties</h4>
-									</td>
-									<td>
-										<code>sideways</code>
-									</td>
-									<td>
-										<code>sideways</code>
-									</td>
-									<td>
-										<code>sideways</code>
-									</td>
-								</tr>
-								<tr>
-									<td>
-										<code>-epub-text-orientation</code>
-									</td>
-									<td>
-										<code>mixed</code>
-									</td>
-									<td>
-										<code>mixed</code>
-									</td>
-									<td>
-										<code>mixed</code>
-									</td>
-								</tr>
-								<tr>
-									<td>
-										<code>-epub-writing-mode</code>
-									</td>
-									<td>
-										<code>horizontal-tb</code>
-									</td>
-									<td>
-										<code>horizontal-tb</code>
-									</td>
-									<td>
-										<code>horizontal-tb</code>
-									</td>
-								</tr>
-								<tr>
-									<td>
-										<code>-epub-writing-mode</code>
-									</td>
-									<td>
-										<code>vertical-rl</code>
-									</td>
-									<td>
-										<code>vertical-rl</code>
-									</td>
-									<td>
-										<code>vertical-rl</code>
-									</td>
-								</tr>
-								<tr>
-									<td>
-										<code>-epub-writing-mode</code>
-									</td>
-									<td>
-										<code>vertical-lr</code>
-									</td>
-									<td>
-										<code>vertical-lr</code>
-									</td>
-									<td>
-										<code>vertical-lr</code>
-									</td>
-								</tr>
-								<tr>
-									<td>
-										<code>-epub-text-combine</code>
-										<span aria-label="deprecated">*</span>
-									</td>
-									<td>
-										<code>-epub-text-combine-horizontal: none</code>
-									</td>
-									<td>
-										<code>none</code>
-									</td>
-									<td>
-										<code>text-combine-upright: none</code>
-									</td>
-								</tr>
-								<tr>
-									<td>
-										<code>-epub-text-combine</code>
-										<span aria-label="deprecated">*</span>
-									</td>
-									<td>
-										<code>-epub-text-combine-horizontal: all</code>
-									</td>
-									<td>
-										<code>horizontal</code>
-									</td>
-									<td>
-										<code>text-combine-upright: all</code>
-									</td>
-								</tr>
-								<tr>
-									<td>
-										<code>-epub-text-combine</code>
-										<span aria-label="deprecated">*</span>
-									</td>
-									<td>Error</td>
-									<td>
-										<code>horizontal &lt;number&gt;</code>
-									</td>
-									<td>
-										<code>text-combine-upright: digits &lt;number&gt;</code>
-									</td>
-								</tr>
-							</tbody>
-						</table>
+						<p>This section describes the <code>-epub-</code> prefixed properties for
+							[[CSS-Writing-Modes-3]].</p>
+
+						<section id="sec-css-prefixed-writing-modes-text-orientation">
+							<h6>The <code>-epub-text-orientation</code> Property</h6>
+							<p>This is a prefixed version of the CSS <code>text-orientation</code> property.</p>
+							<table class="def propdef">
```

Comment:
For this table (and the others, or as some sort of an introductory remark) it should be made clear that, unless otherwise noted, the semantics of a property value for the prefixed version is identical to what CSS specifies (is that correct?). This is stated in, say, the `-epub-writing-mode` property, but not everywhere. In the absence of this the spec is unclear what the meaning of, say, the `vertical-rl` value means

Reply:
That's pretty much the very definition of a prefixed property. I have tried to be very clear about any differences in behavior from current CSS specs. Should I construct larger tables for each property, to explicitly identify that `-epub-writing-mode: vertical-rl` means exactly the same thing as `writing-mode: vertical-rl`?

Reply:
I've added a sentence to the opening para saying that prefixed properties behave the same as their unprefixed equivalent unless otherwise noted. Does this address your concerns?

Reply:
> I've added a sentence to the opening para saying that prefixed properties behave the same as their unprefixed equivalent unless otherwise noted. Does this address your concerns?

Yes! Thx